core/src/main/java/com/google/cloud/sql/CredentialFactory.java

```diff
@@ -26,4 +27,8 @@ public interface CredentialFactory {
   String CREDENTIAL_FACTORY_PROPERTY = "cloudSql.socketFactory.credentialFactory";
 
   HttpRequestInitializer create();
+
+  default CredentialFactory withDelegates(List<String> delegates) {
+    return this;
```

Comment:
I assume this makes the change non-breaking?

Can we write a test that uses the interface as it was, to confirm that this change doesn't break it?

Reply:
Yes, but I think there may be a better way to handle the "with delegates" issue than adding a new method to CredentialFactory. Let me try again on this one.